sdk/cognitiveservices/Vision.Face/tests/SessionRecords/FaceDetectionTests/FaceDetection03WithAttributes.json

```diff
@@ -23,168 +23,35 @@
-    },
-    \"recognitionModel\": \"recognition_04\"
-  }
-]",
+      "ResponseBody": "[\r\n  {\r\n    \"faceId\": \"eee5540a-63f5-4e33-b63d-7c747c433709\",\r\n    \"faceRectangle\": {\r\n      \"top\": 185,\r\n      \"left\": 519,\r\n      \"width\": 132,\r\n      \"height\": 166\r\n    },\r\n    \"faceLandmarks\": {\r\n      \"pupilLeft\": {\r\n        \"x\": 577.4,\r\n        \"y\": 245.4\r\n      },\r\n      \"pupilRight\": {\r\n        \"x\": 625.5,\r\n        \"y\": 230.2\r\n      },\r\n      \"noseTip\": {\r\n        \"x\": 625.7,\r\n        \"y\": 261.8\r\n      },\r\n      \"mouthLeft\": {\r\n        \"x\": 603.6,\r\n        \"y\": 304.2\r\n      },\r\n      \"mouthRight\": {\r\n        \"x\": 641.1,\r\n        \"y\": 292.2\r\n      },\r\n      \"eyebrowLeftOuter\": {\r\n        \"x\": 552.1,\r\n        \"y\": 242.9\r\n      },\r\n      \"eyebrowLeftInner\": {\r\n        \"x\": 592.9,\r\n        \"y\": 227.1\r\n      },\r\n      \"eyeLeftOuter\": {\r\n        \"x\": 568.8,\r\n        \"y\": 248.6\r\n      },\r\n      \"eyeLeftTop\": {\r\n        \"x\": 577.0,\r\n        \"y\": 242.3\r\n      },\r\n      \"eyeLeftBottom\": {\r\n        \"x\": 578.4,\r\n        \"y\": 247.9\r\n      },\r\n      \"eyeLeftInner\": {\r\n        \"x\": 585.5,\r\n        \"y\": 242.8\r\n      },\r\n      \"eyebrowRightInner\": {\r\n        \"x\": 613.2,\r\n        \"y\": 221.2\r\n      },\r\n      \"eyebrowRightOuter\": {\r\n        \"x\": 635.2,\r\n        \"y\": 212.2\r\n      },\r\n      \"eyeRightInner\": {\r\n        \"x\": 618.5,\r\n        \"y\": 232.8\r\n      },\r\n      \"eyeRightTop\": {\r\n        \"x\": 624.9,\r\n        \"y\": 227.1\r\n      },\r\n      \"eyeRightBottom\": {\r\n        \"x\": 626.2,\r\n        \"y\": 232.3\r\n      },\r\n      \"eyeRightOuter\": {\r\n        \"x\": 632.3,\r\n        \"y\": 228.4\r\n      },\r\n      \"noseRootLeft\": {\r\n        \"x\": 598.8,\r\n        \"y\": 240.0\r\n      },\r\n      \"noseRootRight\": {\r\n        \"x\": 614.0,\r\n        \"y\": 235.1\r\n      },\r\n      \"noseLeftAlarTop\": {\r\n        \"x\": 603.1,\r\n        \"y\": 260.7\r\n      },\r\n      \"noseRightAlarTop\": {\r\n        \"x\": 626.3,\r\n        \"y\": 251.4\r\n      },\r\n      \"noseLeftAlarOutTip\": {\r\n        \"x\": 601.5,\r\n        \"y\": 274.1\r\n      },\r\n      \"noseRightAlarOutTip\": {\r\n        \"x\": 634.1,\r\n        \"y\": 262.3\r\n      },\r\n      \"upperLipTop\": {\r\n        \"x\": 627.8,\r\n        \"y\": 288.2\r\n      },\r\n      \"upperLipBottom\": {\r\n        \"x\": 628.2,\r\n        \"y\": 293.5\r\n      },\r\n      \"underLipTop\": {\r\n        \"x\": 629.0,\r\n        \"y\": 296.4\r\n      },\r\n      \"underLipBottom\": {\r\n        \"x\": 630.9,\r\n        \"y\": 303.6\r\n      }\r\n    },\r\n    \"faceAttributes\": {\r\n      \"mask\": {\r\n        \"type\": \"noMask\",\r\n        \"noseAndMouthCovered\": false\r\n      }\r\n    },\r\n    \"recognitionModel\": \"recognition_04\"\r\n  }\r\n]",
```

Comment:
[nit] can this be formatted as before?

Reply:
As this is generated by SessionRecorder and is only used to feed the mock http calls, I think we can keep it in the original generated form.